docs/source/index.rst

```diff
@@ -15,6 +15,9 @@ This project is still very much a work in progress, but I hope that the
 information here will make it easier for newcomers to get started using
 ``Manim``.
 
+.. tip:: All content of the docs are under MIT license. Especially for the examples you encounter:
+Feel free to use this code in your own projects!
+
```

Comment:
Directives (`::`) should usually be written with a newline after `::`, followed by an indented block.

```suggestion
.. tip:: 

    All content of the docs is licensed under the MIT license. Especially for the examples
    you encounter: Feel free to use this code in your own projects!

```